apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions.tsx

```diff
@@ -122,6 +172,163 @@ const SupportAction = ({ show }: SupportActionProps) => {
+  application: Application;
+}
+const CopyApplicationIdAction = ({
+  show,
+  application,
+}: CopyApplicationIdActionProps) => {
+  const intl = useIntl();
+  const [linkCopied, setLinkCopied] = React.useState<boolean>(false);
+  if (!show) {
+    return null;
+  }
+  const jobTitle = getFullPoolTitleHtml(intl, application.pool) && "";
+  return (
+    <Button
+      mode="inline"
+      data-h2-color="base(black.light)"
+      data-h2-font-size="base(caption)"
+      icon={linkCopied ? CheckIcon : undefined}
+      onClick={() => {
+        navigator.clipboard.writeText(application.id);
+        setLinkCopied(true);
+      }}
+      aria-label={
+        linkCopied
+          ? intl.formatMessage(
+              {
+                defaultMessage: "Copy your {title} application's ID",
+                id: "WJSVBr",
+                description: "Button text to copy a specific application ID",
+              },
+              {
+                title: jobTitle,
+              },
+            )
+          : intl.formatMessage(
+              {
+                defaultMessage: "Your {title} application's ID copied",
+                id: "ftg8sT",
+                description:
+                  "Button text to indicate that a specific application's ID has been copied",
+              },
+              {
+                title: jobTitle,
+              },
+            )
+      }
+    >
+      {linkCopied
+        ? intl.formatMessage({
+            defaultMessage: "Application ID copied",
+            id: "gBAz/G",
+            description:
+              "Button text to indicate that a specific application's ID has been copied",
+          })
+        : intl.formatMessage({
+            defaultMessage: "Copy application ID",
+            id: "rvoNoQ",
+            description: "Button text to copy a specific application ID",
+          })}
+    </Button>
+  );
+};
+export interface VisitResumeActionProps extends ActionProps {
+  userID: string;
+  application: Application;
+}
+const VisitResumeAction = ({
+  show,
+  userID,
+  application,
+}: VisitResumeActionProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const jobTitle = getFullPoolTitleHtml(intl, application.pool) && "";
+
+  if (!show) {
+    return null;
+  }
+
+  const recruitmentSectionUrl = paths.resumeAndRecruitment(userID, {
+    applicationId: application.id,
+    section: PAGE_SECTION_ID.QUALIFIED_RECRUITMENT_PROCESSES,
+  });
+
+  return (
+    <Link
+      href={recruitmentSectionUrl}
+      mode="inline"
+      data-h2-color="base(black.light)"
+      data-h2-font-size="base(caption)"
+      aria-label={intl.formatMessage(
+        {
+          defaultMessage: "Visit the {title} recruitment on your résumé",
+          id: "pqsnpk",
+          description: "Link text to direct a user to the Résumé page",
+        },
+        {
+          title: jobTitle,
+        },
+      )}
+    >
+      {intl.formatMessage({
+        defaultMessage: "Visit résumé",
```

Comment:
Shouldn't this be something like "Visit résumé page"? The French translation, "Consulter le cv ", makes it seem like a user would be viewing an actual document rather than a page.

Reply:
hmm not sure. we only use résumé in the application. Its similar to Review job ad.

Reply:
Leave as is.